Foundation/CGFloat.swift

```diff
@@ -615,174 +619,178 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
 // tgmath
 //===----------------------------------------------------------------------===//
 
+#if os(Android)
+import Glibc
+#endif
+
 @_transparent
 public func acos(_ x: CGFloat) -> CGFloat {
-    return CGFloat(acos(x.native))
+    return CGFloat(acos(x.double))
```

Comment:
D'oh, of course. So that means these functions never did compile on 32-bit platforms.

I'm guessing you didn't mean to add `double` as part of the `CGFloat` public API, though. OTOH, I suppose it ought to be `public` if you're invoking from an `@_transparent` function. Would maybe `public var _doubleValue` be appropriate? And shouldn't it be `@_transparent` itself?